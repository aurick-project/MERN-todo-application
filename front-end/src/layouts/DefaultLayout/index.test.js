import React from 'react';
import renderer from 'react-test-renderer';

// import { shallow } from 'enzyme';

import { DefaultLayout } from './index';

// describe('AddTaskForm', () => {
//   it('should render correctly in "debug" mode', () => {
//     const component = shallow(<AddTaskForm debug />);
//     expect(component).toMatchSnapshot();
//   });
// })

// ref: https://jestjs.io/docs/tutorial-react
test('DefaultLayout', () => {
  const component = renderer.create(<DefaultLayout>Default Layout</DefaultLayout>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
