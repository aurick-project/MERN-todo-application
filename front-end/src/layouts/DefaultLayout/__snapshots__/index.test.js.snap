// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultLayout 1`] = `
<div
  className="MuiContainer-root container-0-3-1 MuiContainer-maxWidthSm"
>
  <h4
    className="MuiTypography-root title-0-3-2 MuiTypography-h4 MuiTypography-alignCenter MuiTypography-displayBlock"
  >
    Todo App
  </h4>
  Default Layout
</div>
`;
